src/zenml/config/docker_settings.py

```diff
@@ -239,6 +241,7 @@ class DockerSettings(BaseSettings):
     local_project_install_command: Optional[str] = None
     apt_packages: List[str] = []
     environment: Dict[str, Any] = {}
+    post_environment: Dict[str, Any] = {}
```

Comment:
Maybe something like `runtime_environment`? `post_environment` is so unspecific to me, post what?

Reply:
agreed on this